### PR TITLE
Heap's Algorithm for generating permutations for N objects

### PR DESCRIPTION
**Pull request by kevinburchfield**

### CODE REVIEW

File: math/permutation/heaps.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+			}
+		}
+	}
+	recursiveGenerate(permutations, n, elementSet)
+	out <- permutations
+}
+
+func GenerateElementSet(out chan []string, n int) {
+	elementSet := make([]string, n)
+	for i, _ := range elementSet {
```
Comment (brayo-pip) [inline math/permutation/heaps.go]: just remove the _

**Review by brayo-pip — COMMENTED** [review]
remove the blank variable _ on line  39

**Comment by kevinburchfield** [comment]
> remove the blank variable _ on line 39

I've done that in [this commit](https://github.com/TheAlgorithms/Go/pull/301/commits/ac5590df6e12864d54d849d319978ecb40ac37e6). Thanks for the review!

**Comment by kevinburchfield** [comment]
@brayo-pip Could you confirm that I removed the blank variable and merge?

**Pull request merged**